### PR TITLE
add dump_tf to coda_process

### PR DESCRIPTION
**Pull request by cmr**
helps when debugging integration tests that don't involve a worker crashing.

### CODE REVIEW

File: src/app/cli/src/coda_worker_testnet.ml [hunk 1]
```diff
@@ -149,10 +149,22 @@ let start_prefix_check logger workers events testnet ~acceptable_delay =
       !"lengths: %{sexp: int list} shared_prefix: %{sexp: string option} \
         shared_prefix_age: %d"
       lengths newest_shared shared_prefix_age ;
-    if not (shared_prefix_age <= 5) then (
-      Logger.fatal logger ~module_:__MODULE__ ~location:__LOC__
-        "prefix too old" ;
-      ignore (exit 1) ) ;
+    let chains_list = Array.to_list chains in
```
Comment (enolan) [inline src/app/cli/src/coda_worker_testnet.ml]: this stuff is supposed to be in a different commit right?

Comment (cmr) [reply src/app/cli/src/coda_worker_testnet.ml]: no, this is just adding metadata to the log.

Comment (enolan) [reply src/app/cli/src/coda_worker_testnet.ml]: oh I see. i got confused since you were modifying the shared prefix test as part of other work
